service/history/timerQueueProcessor.go

```diff
@@ -787,9 +787,25 @@ Update_History_Loop:
 			return nil
 		}
 
+		// Generate a transfer task to delete workflow execution
```

Comment:
Looks like this is a clone of code in engine.  Can we we refactor so we don't have multiple copies of the same code?